### PR TITLE
Reconciliation of data paths in notebooks with latest PyKOALA version

### PR DESCRIPTION
**Pull request by miguelglezb**
I had to make a fork of this repository since I don't have maintainers' privileges. I corrected all the data paths and copied the fits files necessary to run the notebooks. As soon as this is reviewed and approved, we can move to the next step in our to-do list.

### CODE REVIEW

**Comment by PabloCorcho** [comment]
I see that the new structure for the data (separated into different gratings) is not used in this PR, and it switches everything back to the original version. I think we should make use of the new one, which provides a much cleaner way of dealing with the data

File: tutorials/reduction_sequence_science.ipynb [hunk 1]
```diff
@@ -147,12 +112,16 @@
...
+    "path_to_data = 'data'\n",
+    "\n",
     "adr_set = []\n",
     "\n",
     "#for i in [31, 32, 33]:\n",
     "for i in [34, 35, 36]:\n",
-    "    filename = os.path.join(\"data\", \"koala\", grating,\n",
-    "                            f\"27feb{aaomega_gratings[grating]}00{i}red.fits\")\n",
+    "    filename = f\"data/koala/385R/27feb200{i}red.fits\"\n",
     "    rss = koala_rss(filename)\n",
```
Comment (PabloCorcho) [inline tutorials/reduction_sequence_science.ipynb:125]: I believe making use of the `os` library for dealing with the paths is always much better than hardcoding the paths manually, why did you change that?

File: tutorials/reduction_sequence_science.ipynb [hunk 2]
```diff
@@ -294,7 +261,7 @@
     "from pykoala.corrections.flux_calibration import FluxCalibration\n",
     "from pykoala.plotting.qc_plot import qc_cube\n",
     "\n",
-    "fcal = FluxCalibration(path_to_response=os.path.join(\"products\", \"koala\", grating, 'response_HILT600_transfer_function.dat'), verbose=True)\n",
+    "fcal = FluxCalibration(path_to_response='products/response_HILT600_transfer_function.dat', verbose=True)\n",
     "\n",
     "plt.figure()\n",
```
Comment (PabloCorcho) [inline tutorials/reduction_sequence_science.ipynb:266]: Same here. This basically breaks the way the products are stored in terms of the grating used.

**Comment by PabloCorcho** [comment]
In general, I think the way the data is organized in the new version is much more intuitive for the user, allowing them to switch between gratings effortlessly. Unfortunately, this is not compatible with the previous version that was used during the tutorials. I´d argue that it is worth to make the effort to adopt the new data structure.

**Comment by miguelglezb** [comment]
Right, I just restored the deleted lines and modified the data and products storage structure locally. I will close this pull request and make a new one with the new configuration.
